### ISSUE TITLE
Feature Request: store by default in .vscode/.history folder instead of .history

### ISSUE THREAD

**Issue by hervelemeur** [zabel-xyz/local-history#130]
So when adding it to .gitignore we can put `.vscode` instead of `.history` (more common)

**Comment by robross0606** [comment]
I already requested this (#110).  There are _all_ sorts of other problems with storing it in the root of the workspace.  To name a few:
- Test infrastructures like jest automatically find history files and try to run tests on them.
- Linting infrastructure like eslint automatically finds .history and tries to lint them.
- VSCode itself turns up the .history folder in search results.

Do I need to go on?  This folder should _never_ have been put in the root of the workspace to begin with.  The last time I suggested a change, it was shot down.

**Comment by ignlg** [comment]
tl;dr: There is a configuration value to select the .history folder path.

Is not it enough adding it to `.gitignore`, `.eslintignore`, and such? We did it and never had these issues.

It is like `.vscode` or `coverage` or `dist` or any other root folder that should be ignored. IMO, even being a VScode extension, this folder could have a different treatment than `.vscode` files that are, mostly, local configurations. 

If this issue goes forward, I would suggest to go beyond this and add an option to select the folder location instead, this would do the trick for everyone, wouldn't it?

Addendum: It is already done, found at https://github.com/zabel-xyz/local-history/issues/129

**Comment by pauloch8** [comment]
I think this feature is important, since I can´t force my teammates to check stashed files before commit, I won´t suggest this extensions to them.

**Comment by robross0606** [comment]
Agreed, this is horribly annoying because _every_ project you open has this problem once the extension is installed.  The author is basically forcing people to start encoding the use of their plugin into files which are _committed_ to repos (`.gitignore`, `.eslintignore`, etc.).  One user may use this extension while 150 other people don't.  This needs to be a system/user setting.

**Comment by adamgordonbell** [comment]
Git has a global gitignore:
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

**Comment by robross0606** [comment]
> Git has a global gitignore:
> https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Git is not the only problem this causes.  I've listed them in the ticket itself.  There are more than just the three listed too.  How many different things have to go wrong before someone can admit the design is just flawed?

**Comment by robross0606** [comment]
Here's yet _another_ example of where this causes problems.  If you're not careful, `npm publish` will _also_ include your `.history` folder.

**Comment by ganomi46** [comment]
This is needed. The .history directory messes up with search, git, etc.
